recipe/meta.yaml

```diff
@@ -1,13 +1,15 @@
-{% set version = "0.1" %}
+{% set name = "osmnx" %}
+{% set version = "0.2.1" %}
+{% set md5 = "83614b39ca948bdbc1b4373b478d0372" %}
 
 package:
-  name: osmnx
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: osmnx-{{ version }}.tar.gz
-  url: https://github.com/gboeing/osmnx/archive/v{{ version }}.tar.gz
-  sha256: 9fda466d434022b55f41f0511c99cdf59bf5119630caa5263a315f2d3ab3524c
```

Comment:
@gboeing some of the changes here are OK but I prefer:
- sha256 because it is saver
- GitHub tarballs because they are "closer" to the source (for some libs that may contain LICENSE files that are usually missing in the PyPI tarballs)
- jinja variables only for the version b/c it is easier to update and copy-n-paste the link in case we need to download the source manually to check anything.

With that said I am OK accepting your changes since you are actively maintaining this feedstock too, and you are the original author of the package :smile:

I thought I should let you know my reasons for that format though :wink: